modules/virt-creating-blank-disk-datavolumes.adoc

```diff
@@ -26,14 +26,12 @@ metadata:
   name: blank-image-datavolume
 spec:
   source:
-      blank: {}
-  pvc:
-    storageClassName: "hostpath" <1>
-    accessModes:
-      - ReadWriteOnce
+    blank: {}
+  storage:
     resources:
       requests:
-        storage: 500Mi
+        storage: <2Gi>
+    storageClassName: "<storage_class>" <1>
```

Comment:
@awels I might as well remove the optional storage class line (leaving the size). If the user is creating a blank data volume, they probably don't care if it uses the default storage class.

Reply:
That is always the question, how much do we show and do we not show. In case someone does want to pick their storage class, there is no example. But too many similar examples can be confusing.

Reply:
OK. I can revert the storage class option.